examples/srd_y10_3x2.yaml

```diff
@@ -255,6 +255,7 @@ analyze:
       {% endfor %}
 
 postprocess:
-  input: *fisher_fname
-  latex_table: srd_y10.tex
-  triangle_plot: srd_y10.pdf
+  latex_table: output_fishery10/latex_table.tex
+  triangle_plot: output_fishery10/triangle_plot.pdf
+  pairplots: [(w0, wa), (omega_c, sigma8)]
+  CL: 0.68
```

Comment:
Ok, but if you change this I think the FoM won't change, it will still be 68%. Otherwise we need to add the Fisher scale factor from the chi2 distribution. Perhaps the CL could be passed explicitly to get_FoM to get the FoM appropriate for that CL and avoid inconsistencies.